draft-irtf-cfrg-frost.md

```diff
@@ -695,6 +695,27 @@ The value of the contextString parameter is "FROST-RISTRETTO255-SHA512".
     the output to a Scalar as described in {{!RISTRETTO, Section 4.4}}.
   - H3(m): Implemented by computing H(contextString \|\| "digest" \|\| m).
 
+## FROST(edwards448, SHAKE256)
+
+This ciphersuite uses edwards448 for the Group and SHA256 for the Hash function `H`
+meant to produce signatures indistinguishable from Ed448 as specified in {{!RFC8032}}.
+The value of the contextString parameter is empty.
+
+- Group: ed448 {{!RFC8032}}
+  - SerializeElement: Implemented as specified in {{!RFC8032, Section 5.2.2}}.
+  - DeserializeElement: Implemented as specified in {{!RFC8032, Section 5.2.3}}.
+- Hash (`H`): SHAKE256, and Nh = 117.
+  - H1(m): Implemented by computing H("rho" || m), interpreting the lower
+    57 bytes as a little-endian integer, and reducing the resulting integer modulo
+    L = 2^446 - 13818066809895115352007386748515426880336692474882178609894547503885.
+  - H2(m): Implemented by computing H(m), interpreting the lower 57 bytes
+    as a little-endian integer, and reducing the resulting integer modulo
+    L = 2^446 - 13818066809895115352007386748515426880336692474882178609894547503885.
```

Comment:
This is also to retain compat with Ed448 signatures?

Reply:
:)